eland/ml/imported_ml_model.py

```diff
@@ -59,14 +63,28 @@ class ImportedMLModel(MLModel):
         - sklearn.tree.DecisionTreeRegressor
         - sklearn.ensemble.RandomForestRegressor
         - sklearn.ensemble.RandomForestClassifier
+        - lightgbm.LGBMRegressor
+            - Only the following objectives are supported
+                - "regression"
+                - "regression_l1"
+                - "huber"
+                - "fair"
+                - "quantile"
+                - "mape"
+            - The following booster types are supported
+                - "gbdt"
+                - "rf"
+                - "dart"
+                - "goss"
+            - Categorical fields are expected to already be processed
```

Comment:
I wonder if it makes sense to have this caveat at the top of the section so it isn't visually mistaken for a list element?